snippets/odoo-python-snippets.cson

```diff
@@ -93,61 +93,62 @@
 	'Compute method':
 		'prefix': 'oo_method_compute'
 		'body': """
-@api.one
-@api.depends('$1', ...)
-def _compute_field(self):
-	self.field = $2
-		"""
+			@api.multi
```

Comment:
I know, that's why I change it. It makes no sense to have such ugly indentation if CSON ignores it.